app/controllers/automated_tests_controller.rb

```diff
@@ -51,6 +54,15 @@ def update
             new_script.original_filename)
           File.open(assignment_tests_path, 'w') { |f| f.write new_script.read }
         end
+
+        unless new_support_file.nil?
+          assignment_tests_path = File.join(
+            MarkusConfigurator.markus_config_automated_tests_repository,
+            @assignment.repository_folder,
+            new_support_file.original_filename)
+          File.open(assignment_tests_path, 'w'){ 
+	  |f| f.write new_support_file.read }

```

Comment:
Expression at 64, 38 should be on its own line.<br>Block argument expression is not on the same line as the block start.<br>Tab detected.